db/schema1.cql

```diff
@@ -0,0 +1,24 @@
+DROP KEYSPACE IF EXISTS donutstore;
+
+CREATE KEYSPACE IF NOT EXISTS donutstore WITH replication = {'class': 'SimpleStrategy', 'replication_factor': '1'};
+
+CREATE TABLE donutstore.reviews (
```

Comment:
Why is the keyspace and table name `donutstore`? I don't see how this name is relevant to your data.